services/FormsService.ts

```diff
@@ -27,6 +27,19 @@ export class FormsService extends ApiService {
     );
   }
 
+  async downloadTraineeLtftPdf(id: String) {
+    let requestConfig: AxiosRequestConfig<String> = {
+      headers: {
+        Accept: "application/pdf"
+      },
+      responseType: "blob"
+    };
+    return this.axiosInstance.get<String, AxiosResponse<Blob>>(
```

Comment:
3. I think the first parameter of `axiosInstance.get` should be type `Blob` not `string` as this is the expected res data type.